### DISCUSSION TITLE
Add graph view for LangGraph traces

### DISCUSSION THREAD

**Discussion by hassiebp** [langfuse/langfuse#3409] [Ideas]
### Describe the feature or potential improvement

**Current situation**
LangGraph executions are traced in the same sequential order as the nodes are executed. Also, each execution carries a noisy spans that hold shallow information and deepen the nesting hierarchy unnecessarily.

**Feature request**
Parse the Langgraph execution trace into a graph view. Additionally, remove unnecessary spans to flatten the hierarchy.

**Comment by marcklingen** [comment]
Related: https://github.com/orgs/langfuse/discussions/2669

**Comment by marcklingen** [comment]
Completed, changelog: https://langfuse.com/changelog/2025-02-14-trace-graph-view
